tck/src/main/java/org/eclipse/microprofile/lra/tck/service/LRAMetricType.java

```diff
@@ -20,10 +20,16 @@
 package org.eclipse.microprofile.lra.tck.service;
 
 public enum LRAMetricType {
-    
-    COMPENSATE,
-    COMPLETE,
-    STATUS,
-    FORGET,
-    NESTED
+    // LRA statistics
+    Closed, // an LRA that closed
+    FailedToClose, // an LRA that failed to close
+    Cancelled, // an LRA that was cancelled
+    FailedToCancel, // an LRA that failed to cancel
```

Comment:
this is again kind of subjective, but can we keep the same code style for enum values please? I think either make Complete, Compensate,... or CLOSED, FAILED_TO_CLOSE,...

Reply:
Agreed, the literals are meant to match up with LRAStatus enums